client.go

```diff
@@ -1318,8 +1326,8 @@ func uriPathToURL(uri *URI, path string) url.URL {
 
 func nodePathToURL(node *Node, path string) url.URL {
 	return url.URL{
-		Scheme: node.Scheme,
-		Host:   node.Host,
+		Scheme: node.URI.Scheme(),
```

Comment:
I'll answer my own question a bit: we should probably be calling these URLs. The key thing is the inclusion of a scheme which makes this a locator rather than just an identifier. URLs are a subset of URIs, so URI is not technically incorrect, but URL is more specific. Good article here: https://danielmiessler.com/study/url-uri/